### DISCUSSION TITLE
sqlite adapter - WITHOUT ROWID table option

### DISCUSSION THREAD

**Discussion by loranger32** [jeremyevans/sequel#2125] [General]
Hi,

Using the sqlite adapter, I wanted to create a table with the `WITHOUT ROWID` option.

After some digging in the doc and the code, it seems that the only option Sequel supports is `STRICT`  (lib/sequel/adapters/shared/sqlite.rb : lines 353-355).

I've looked in the Google Group Sequel talk and the Github issues, pull requests and discussions but couldn't find any discussion about the `WITHOUT ROWID` option.

I've come up with a possible solution, but before submitting a PR I wanted to know if it's something you would consider (or if I just overlooked something).

Thanks for this awesome library.

Laurent

**Comment by jeremyevans** [comment]
Supporting `WITHOUT ROWID` seems fine to me, please submit a PR for it.  Please make sure it includes SQLite adapter tests.
